modules/system/classes/PackageJson.php

```diff
@@ -0,0 +1,263 @@
+<?php
+
+namespace System\Classes;
+
+use InvalidArgumentException;
+use RuntimeException;
+use Winter\Storm\Support\Facades\File;
+
+/**
+ * Php based interface for interacting with package.json files. This allows for modification of deps, devDeps, package
+ * name and workspaces.
+ *
+ * @package winter\wn-system-module
+ * @author Jack Wilkinson <me@jackwilky.com>
+ * @author Winter CMS
+ */
+class PackageJson
```

Comment:
I'm not sure people would need a single package to do both, but it's quite small so might not hurt.

Reply:
This is kinda tied to our commands and usage though, is it not? All of that exists in the System module. I don't know how much use it would be in either the Config Writer package or Storm.

Reply:
At the moment it's only being used by Asset commands, it could be used later but until we need it somewhere else maybe we leave it be?

Reply:
Yeah, I feel it probably should remain here unless there's a compelling use case outside of the CMS.